contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -74,6 +76,7 @@ const (
 	defaultMesosUser         = "root" // should have privs to execute docker and iptables commands
 	defaultReconcileInterval = 300    // 5m default task reconciliation interval
 	defaultReconcileCooldown = 15 * time.Second
+	defaultNodeRelistPeriod  = 5 * time.Minute
```

Comment:
I think I misread this before as `5 * time.Second` instead of the `5 * time.Minute` that's actually here. This is fine as-is.